_includes/navigation.html

```diff
@@ -15,50 +15,56 @@ <h2 class="pldoc-heading pldoc-nav-title hd-6">Elements</h2>
     </ul>
 </nav>
 
-<nav class="pldoc-nav pldoc-nav-documentation pldoc-nav-external" id="pldoc-nav-documentation">
-    <h2 class="pldoc-heading pldoc-nav-title hd-6">Documentation</h2>
-    <ul class="pldoc-list nav-list list-styleguides">
+<nav class="pldoc-nav pldoc-nav-documentation pldoc-nav-external" id="pldoc-nav-documentation" role="navigation" aria-label="Documentation">
+    <h2 class="pldoc-heading pldoc-nav-title hd-6" id="heading-documentation">Documentation</h2>
+    <ul class="pldoc-list nav-list list-styleguides" aria-labelledby="heading-documentation">
         <li class="nav-list-item">
             <a class="pldoc-link" rel="external" href="https://github.com/edx/ux-pattern-library/wiki/Styleguide:-General">
                 <span class="pldoc-icon" aria-hidden="true">&#65514;</span>
                 General Styleguide
+                <span class="sr-only">Link opens in a new window</span>
```

Comment:
@clrux, I'm thinking of the behavior as a heads up - the content can be read without it, but that note about the new window/tab is supplemental for context purposes. The parens help distinguish the link label from the contextual help. 

Its a lot like "Thai Peanut Chicken with Brown Rice (contains tasty tasty nuts)" vs. "Thai Peanut Chicken with Brown Rice contains tasty tasty nuts.". To me the latter sounds like a bad asian restaurant menu with grammatical errors.